pool_automation/roles/aws_manage/library/stateful_set.py

```diff
@@ -179,11 +227,12 @@ def _host_info(inst):
 
     valid_region_ids = valid_instances(regions, count)
 
-    for region in AWS_REGIONS:
+    for region in AWS_REGIONS.keys():
```

Comment:
Python iterates dictionaries by keys by default anyway, so there is no real need in specifying `keys` except for additional clarity.